content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,103 @@
+
+---
+title: "4.4.0 release notes"
+linkTitle: "4.4.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+4.4.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### CHT dialogs.
+
+CHT dialogs were updated to align with Material guidelines. This includes updating the visual design / placement of dialogs as well as updating the default text of dialogs as necessary.
+
+{{< figure src="../images/4.4.0-8391-cht-dialogs.png" class="left col-12 col-lg-12" >}}
+<br clear="all">
+
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+
+
+## Highlights
+
+### Modals\'s text
+
+Some modals\'s text have been improved to increase understanding and some keys have been replaced.
+
+- Bulk delete reports
+  - `bulkdelete.confirm.title` used for the modal title when deleting 1 report
+  - [NEW] `bulkdelete.confirm.title.plural` used for the modal title when deleting multiple reports.
+- Edit message group
+  - [NEW] `edit_message_group.modal.title`: used for the modal title
+- Report a bug
+  - [NEW] `feedback.modal.submit`: used for the modal submit button label. 
+- Confirm Navigation - like when exiting a form that hasn't been submitted
+  - [NEW] `confirm.destructive.navigation.title`: used for the modal title
+  - [NEW] `confirm.destructive.navigation.submit`: used for the modal submit button label.
+- Training cards
+  - [NEW] `training_cards.confirm.title`: used for the modal title
+- Confirm verifying a report
+  - [NEW] `confirm.verification.title`: used for the modal title
+  - [NEW] `confirm.verification.submit`: used for the modal submit button label.
+
+We have provided translations for all of these in the CHT Core supported languages (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili)
```

Comment:
```suggestion
As part of this work, some translation keys changed. We have provided translations for all of the languages officially supported by CHT (English, Spanish, French, Indonesian, Hindi, Nepali, Bambara, Swahili). If you have a custom language and have any of these features, you will need to translate these in Admin app > Display > Translations tab. 

- Bulk delete reports dialog
  - `bulkdelete.confirm.title`: used for the modal title when deleting one report
  - [NEW] `bulkdelete.confirm.title.plural`: used for the modal title when deleting multiple reports.
- Edit SMS message group dialog
  - [NEW] `edit_message_group.modal.title`: used for the modal title
- Report a bug dialog
  - [NEW] `feedback.modal.submit`: used for the modal submit button label. 
- Navigation when form not saved dialog
  - [NEW] `confirm.destructive.navigation.title`: used for the modal title
  - [NEW] `confirm.destructive.navigation.submit`: used for the modal submit button label.
- Training cards dialog
  - [NEW] `training_cards.confirm.title`: used for the modal title
- Confirmation when verifying a report dialog
  - [NEW] `confirm.verification.title`: used for the modal title
  - [NEW] `confirm.verification.submit`: used for the modal submit button label.
```
I think better move this to line 29 because is about the same work.

Reply:
Done 👍